ExtraCalcFunctions.php

```diff
@@ -34,6 +34,15 @@ public function redcap_every_page_before_render( $project_id )
 		$lastDuration = $project_id === null
 		                ? 0 : ( $this->getProjectSetting( 'calc-values-auto-update-dur' ) ?? 0 );
 		$autoCalcWait = $lastDuration < 90 ? 900 : ( $lastDuration * 10 );
+		$thisIteration = $project_id === null
+		                 ? 0 : ( $this->getProjectSetting( 'calc-values-auto-update-itr' ) ?? 1 );
+		$splitRuns = $project_id === null
+		             ? 0 : ( $this->getProjectSetting( 'calc-values-auto-update-spl' ) ?? 1 );
+		if ( $lastDuration > 600 || $lastDuration === -1 )
+		{
+			$splitRuns++;
```

Comment:
I know generally speaking the direction of travel is always more records >> more calculations to do.  But... if their were performance improvements made, say making a calculated function more efficient, could we decrease the split run count (or does it start again at 1 at start of day or start of user session?)